features/topology-map/plugins/org.opennms.features.topology.plugins.topo.linkd/src/main/java/org/opennms/features/topology/plugins/topo/linkd/internal/providers/EnhancedLinkdTopologyProvider.java

```diff
@@ -404,65 +385,27 @@ public Integer getTargetIfIndex() {
-
-    static final String[] OPER_ADMIN_STATUS = new String[] {
-            "&nbsp;",          //0 (not supported)
-            "Up",              //1
-            "Down",            //2
-            "Testing",         //3
-            "Unknown",         //4
-            "Dormant",         //5
-            "NotPresent",      //6
-            "LowerLayerDown"   //7
-    };
-
-    private LldpLinkDao m_lldpLinkDao;
-    private LldpElementDao m_lldpElementDao;
-    private CdpLinkDao m_cdpLinkDao;
-    private CdpElementDao m_cdpElementDao;
-    private OspfLinkDao m_ospfLinkDao;
-    private IsIsLinkDao m_isisLinkDao;
-    private BridgeBridgeLinkDao m_bridgeBridgeLinkDao;
-    private BridgeMacLinkDao m_bridgeMacLinkDao;
-    private BridgeTopologyDao m_bridgeTopologyDao;
-    private IpNetToMediaDao m_ipNetToMediaDao;
+    protected static Logger LOG = LoggerFactory.getLogger(EnhancedLinkdTopologyProvider.class);
 
     private SelectionAware selectionAwareDelegate = new EnhancedLinkdSelectionAware();
-    
-    public final static String LLDP_EDGE_NAMESPACE = TOPOLOGY_NAMESPACE_LINKD + "::LLDP";
-    public final static String OSPF_EDGE_NAMESPACE = TOPOLOGY_NAMESPACE_LINKD + "::OSPF";
-    public final static String ISIS_EDGE_NAMESPACE = TOPOLOGY_NAMESPACE_LINKD + "::ISIS";
-    public final static String BRIDGE_EDGE_NAMESPACE = TOPOLOGY_NAMESPACE_LINKD + "::BRIDGE";
-    public final static String CDP_EDGE_NAMESPACE = TOPOLOGY_NAMESPACE_LINKD + "::CDP";
+
 
     private final Timer m_loadFullTimer;
     private final Timer m_loadNodesTimer;
     private final Timer m_loadIpInterfacesTimer;
     private final Timer m_loadSnmpInterfacesTimer;
-    private final Timer m_loadIpNetToMediaTimer;
     private final Timer m_loadLldpLinksTimer;
-    private final Timer m_loadOspfLinksTimer;
-    private final Timer m_loadCdpLinksTimer;
-    private final Timer m_loadIsisLinksTimer;
-    private final Timer m_loadBridgeLinksTimer;
     private final Timer m_loadNoLinksTimer;
-    private final Timer m_loadManualLinksTimer;
 
-    public EnhancedLinkdTopologyProvider(MetricRegistry registry) {
+    public EnhancedLinkdTopologyProvider(MetricRegistry registry, String namespace) {
+        super(namespace);
         Objects.requireNonNull(registry);
         m_loadFullTimer = registry.timer(MetricRegistry.name("enlinkd", "load", "full"));
         m_loadNodesTimer = registry.timer(MetricRegistry.name("enlinkd", "load", "nodes"));
         m_loadIpInterfacesTimer = registry.timer(MetricRegistry.name("enlinkd", "load", "ipinterfaces"));
         m_loadSnmpInterfacesTimer = registry.timer(MetricRegistry.name("enlinkd", "load", "snmpinterfaces"));
-        m_loadIpNetToMediaTimer = registry.timer(MetricRegistry.name("enlinkd", "load", "ipnettomedia"));
         m_loadLldpLinksTimer = registry.timer(MetricRegistry.name("enlinkd", "load", "links", "lldp"));
-        m_loadOspfLinksTimer = registry.timer(MetricRegistry.name("enlinkd", "load", "links", "ospf"));
```

Comment:
Can we restore these timers? They give us a nice breakdown of the time spent loading each of the topologies.